.env

```diff
@@ -0,0 +1,3 @@
+SPRING_DATASOURCE_USERNAME=p2team1_user
```

Comment:
.env shouldn't be committed to into version control. Security issue to have passwords for the entire world to see